### ISSUE TITLE
install

### ISSUE THREAD

**Issue by mrbreil** [chefbennyj1/Emby.AlexaController#1]
hello
can you please tell me how to install this.
Thank You

**Comment by chefbennyj1** [comment]
Yes. I'll put a release up in this repo. There is a bit of a setup to get it working. If you are up for it.

**Comment by mrbreil** [comment]
in which year??? ;-)

**Comment by chefbennyj1** [comment]
😆. Fair enough. I'll build one shortly.

**Comment by gohlas** [comment]
Also waiting here 😁
